### ISSUE TITLE
Section Edit for Tab Edits Wrong Block

### ISSUE THREAD

**Issue by Miro-Collas** [LotarProject/dokuwiki-plugin-bootswrapper#91]
Say I have a set of tabs set up, along the lines of this:
```
<tabs>
  * [[#tab-a|A]]
  * [[#tab-b|B]]
  * [[#tab-c|C]]
  * [[#tab-d|D]]
  * [[#tab-e|E]]
  * [[#tab-f|F]]
.... etc
  * [[#tab-z|Z]]
</tabs>
```
I click on, for example, E to view that section, then want to edit it. If I double click in the space, I go into edit on Z.
If I click the section edit right under the tabs, I go into edit on the preceding block, or on the `<tabs> ... </tabs>` block if I have A selected. 
If I click the section edit for E, I go into edit for the Z block, oddly.

In short, the is no way to section edit when using tabs.

**Comment by LotarProject** [comment]
Hi @Miro-Collas,
i have added the support of Edit Button for Pane syntax.

**Comment by Miro-Collas** [comment]
Awesome, thank you! :-)
